### PR TITLE
Remove the "RuntimeError: Timeout during packet send" error generated…

### PR DESCRIPTION
**Pull request by geekguy-wy**
Remove the error generated when a packet send times out. Fixes issue #26.

### CODE REVIEW

**Comment by jerryneedell** [comment]
You can handle this error in your user code with a try/except.  It should not normally occur and I think it should throw an error. 
I’ll be interested in what others think.

**Comment by geekguy-wy** [comment]
Does that error indicate anything other than that the send timed out? Or, does it mean that the send did not finish or that some other condition that needs attention has occurred?

**Comment by jerryneedell** [comment]
Another option is to have send() return a Boolean indicating success or failure. 
That way the user can check the status if the send(). This will impact my pending PR and I would prefer to implement it there and close this PR if you want to go that route.

**Comment by jerryneedell** [comment]
The error indicates some unexpected failure. I have seen them, but they have been rare.  It is not clear to me why you should be seeing them. I’d prefer to investigate the root cause and see if we can until. The original library did not even check. As noted I only added the timeout because I was taught that such loops should have timeouts! You can also see if making the timeout longer helps.

**Comment by jerryneedell** [comment]
The more I think about it, the more i like the idea of having send() return a woolen:
True if successful (no timeout)
False if failed (timed out)
That way the user can check or ignore the status as desired. much like the proposed method of issuing no error. But it give the user the option of reacting without the need for a try/except.

File: adafruit_rfm69.py [hunk 1]
```diff
@@ -727,8 +727,6 @@ def send(self, data, timeout=2., keep_listening=False,
         else:
         # Enter idle mode to stop receiving other packets.
             self.idle()
-        if timed_out:
-            raise RuntimeError('Timeout during packet send')
```
Comment (jerryneedell) [inline adafruit_rfm69.py]: suggest changing this  to;
return not timed_out

Comment (jerryneedell) [reply adafruit_rfm69.py]: also would need to update the doctsring.

Comment (geekguy-wy) [reply adafruit_rfm69.py]: Making the timeout longer does not help.

Changing it to return (not timed_out) is a good alternative and will allow the caller to decide whether it needs further action or not.

Comment (jerryneedell) [reply adafruit_rfm69.py]: would you like to make that change in this PR or would you like me to incorporate it into PR #24

**Comment by jerryneedell** [comment]
This looks good to me -- I have not tested it yet.I can do that later this afternoon.

**Comment by geekguy-wy** [comment]
The third commit updates the doc string.

**Comment by jerryneedell** [comment]
> > Another option is to have send() return a Boolean indicating success or failure.
> > That way the user can check the status if the send(). This will impact my pending PR and I would >prefer to implement it there and close this PR if you want to go that route.
> 
> How does this impact your PR, Jerry?

I'll be able to work with it.  I have to think about it a bit but it may not require any changes. No reason to hold it up for me.

**Comment by jerryneedell** [comment]
@caternuson @brentru any concerns or should I go ahead and merge this?

**Comment by jerryneedell** [comment]
Hearing no objections, I will go head and merge this PR in  a little while.

**Comment by ladyada** [comment]
thanks!

**Pull request merged**